### PR TITLE
add backports.shutil_which package

### PR DESCRIPTION
**Pull request by kain88-de**
needed for #2115

### CODE REVIEW

File: recipes/backports.shutil_which/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - backports
+    - backports.shutil_which
+
+about:
+  home: http://github.com/simplejson/simplejson
```
Comment (jakirkham) [inline recipes/backports.shutil_which/meta.yaml]: Probably need to update this.

**Comment by jakirkham** [comment]
Please also enable CircleCI on your fork of staged-recipes.

xref: https://github.com/conda-forge/status/issues/30

**Comment by kain88-de** [comment]
I'm doing the build now with circleCI enabled.

**Comment by kain88-de** [comment]
btw the circleCI build fails due to the MPI package currently in master.

File: recipes/backports.shutil_which/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - python
+
+test:
+  imports:
+    - backports
+    - backports.shutil_which
+
+about:
+  home: https://github.com/minrk/backports.shutil_which
+  license: MIT
```
Comment (jakirkham) [inline recipes/backports.shutil_which/meta.yaml]: Based on this [note]( https://github.com/minrk/backports.shutil_which/commit/496539f113d3ce2bdf9d737d4f16e176f73c2655 ) it should be MIT, PSF 2.

Comment (kain88-de) [inline recipes/backports.shutil_which/meta.yaml]: thanks I overlooked that.

File: recipes/backports.shutil_which/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+about:
+  home: https://github.com/minrk/backports.shutil_which
+  license: PSF2 & MIT
+  license_file: LICENSE
+  summary: 'Backport of shutil.which from Python 3.3'
+
+extra:
+  recipe-maintainers:
+    - kain88-de
```
Comment (jakirkham) [inline recipes/backports.shutil_which/meta.yaml]: @minrk , any interest in being listed as a maintainer?

Comment (minrk) [inline recipes/backports.shutil_which/meta.yaml]: sure

**Comment by jakirkham** [comment]
Thanks @kain88-de .

**Pull request merged**